Modules/Devices/src/Devices.Infrastructure.Database.SqlServer/Migrations/ApplicationDbContextModelSnapshot.cs

```diff
@@ -170,25 +170,6 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.ToTable("AspNetUsers", (string)null);
                 });
 
-            modelBuilder.Entity("Backbone.Modules.Devices.Domain.Entities.Challenge", b =>

```

Comment:
Why is this change here?

Reply:
I'm not actually sure where this is coming from.
The same thing happened when I merged the changes from main into the identity-deletion and recreated the migrations.

Reply:
The migrations should be fixed now.